src/components/InputAccountNumber.js

```diff
@@ -10,12 +10,14 @@ const SInput = styled(Input)`
 class InputAccountNumber extends Component {
   static propTypes = {
     getRef: PropTypes.func,
-    required: PropTypes.bool
+    required: PropTypes.bool,
+    onChange: PropTypes.func
   };
 
   static defaultProps = {
     getRef: null,
-    required: false
+    required: false,
```

Comment:
do you need to explicitly pass required here? Should be handled within `{ ...props }`